meson_options.txt

```diff
@@ -2,7 +2,7 @@ option('bash_completion', type: 'boolean', value: true, description: 'install ba
 option('dbus-service-dir', type: 'string', value: '', description: 'path to the dbus service dir')
 option('introspection', type: 'boolean', value: true, description: 'build GIR data')
 option('keys-dir', type: 'string', value: '', description: 'path to where gnome-control-center stores its keybindings')
-option('shell-extension', type: 'boolean', value: true, description: 'install the gnome-shell extension')
+option('gnome-shell', type: 'boolean', value: true, description: 'install the gnome-shell extension')
```

Comment:
there are couple of other option names that are bogging me, like "keys-dir" and the completions with underscores when all the others are using hyphens. I wonder how other projects call those

Reply:
Ah, usually everything is using hyphens, but the completions (no idea why, but other projects also do that). I don't have a problem with renaming the options though (or you could do that :)

Reply:
I'll probably do a pass before releasing